docs/UserGuide.md

```diff
@@ -387,7 +384,7 @@ Examples:
 
 ### Deleting an event : `delete event`
 
-Deletes the specified event from the address book.
+This command allow you to delete the specified event from the list of events on TalentHub!
```

Comment:
I think there "allow" should be "allows".

Reply:
Done!